roles/static_files/templates/etc/nginx/sites-available/static_files

```diff
@@ -8,12 +8,15 @@ server {
     listen   8080;
     listen   [::]:8080 default ipv6only=on;  ## listen for ipv6
     port_in_redirect off;
+    expires          -1;
```

Comment:
?? Pre-expired, or infinite?

Reply:
Default is no cache, unless explicitly set elsewhere (which they should all be).

Reply:
I guess that answers my question: -1 is pre-expired, so won't be cached.